ansible_collections/arista/avd/roles/eos_designs/docs/tables/node-type-wan-configuration.md

```diff
@@ -55,14 +55,11 @@
         wan_role: <str; "client" | "server">
 
         # PREVIEW: This key is currently not supported
-        # Override the default CV Pathfinder role.
+        # Configure the transit mode for a WAN client for CV Pathfinder designs
+        # only when the `wan_mode` root key is set to `cv_pathfinder`.
 
-        # This key is used for Pathfinder designs only when the `wan_mode` root
-        # key is set to `cv_pathfinder`.
-
-        # `pathfinder` is only a valid if `wan_role` is `server`.
-        # `edge` and `transit region` are only valid if `wan_role` is `client`.
-        cv_pathfinder_role: <str; "edge" | "transit region" | "pathfinder">
+        # 'zone' is not supported today.
+        cv_pathfinder_transit_mode: <str; "none" | "zone" | "region"; default="none">
```

Comment:
Would it all be simpler if we dropped the "none" value here and just did not have a default value?